src/db/api.js

```diff
@@ -0,0 +1,15 @@
+const SchoolType = require("./models");
+
+const createSchoolType = () => {
```

Comment:
Is this function anywhere?

Reply:
Already not. But I'm going to use it in fixture loader and of course it will be used in the future to insert new types of schools.